### ISSUE TITLE
parallelize-queries parameter 

### ISSUE THREAD

**Issue by ZedZipDev** [Seddryck/NBi#440]
I have changed the config parameter:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<testSuite name="MyTestSuite" xmlns="http://NBi/TestSuite">
  <settings>
    <parallelize-queries>false</parallelize-queries>
  </settings>
```

Then run test and receive: 
ResultSet defined through a query.
ResultSet defined through a query.
**Queries exectued in parallel.**

For  any value of 'parallelize-queries' the log record is the same: Queries exectued in parallel.
By the way: **"exectued"**

**Comment by Seddryck** [comment]
Hi @Oleg26Dev 

This issue rings a bell but I don't remember exactly what and can't find references in GitHub. I'll check but it will wait a bit. 

Anyway, except in really rare cases, there is no good reason to use ```parallelize-queries``` set to ```false```. Why do you want to set this flag off?

**Comment by ZedZipDev** [comment]
When I have both queries to the same server/db and parallelize-queries=false the test works ok. If parallelize-queries=true it gives me error about duplicated keys

**Comment by Seddryck** [comment]
You previously told me that when you set parallelise-queries to false it was not effective and now few days later setting this parameter to false would solve a completely new issue related to duplicate keys. 

What’s exactly your problem? What are you trying to achieve?

What’s your version of NBi? 

Could you post the full test and error message if you have duplicated keys? Are you working on the same project than @win32nipuh?

Are you able to reproduce this behaviour on AdventureWorks ?

**Comment by ZedZipDev** [comment]
Ok, I understand where is the reason of problem.
It is not NBi, thanx!
